Backend/src/main/java/com/onbank/exceptions/advice/ExceptionControllerAdvice.java

```diff
@@ -22,8 +24,16 @@ private void printError(Exception e){
         e.printStackTrace();
     }
 
+    @ExceptionHandler(NoSuchElementException.class)
```

Comment:
Takich rzeczy to nie róbmy. Nadpisujemy obsługę wyjątku, który jest wyrzucany przez Optional.get(). To, że w przypadku naszej usługi chcemy zwracać 404 nie znaczy że zawsze taki scenariusz będzie pożądany. Powinniśmy utworzyć tutaj własny wyjątek np UserNotFoundException i w przypadku nie znalezienia użytkownika wyrzucać go w serwisie i analogiczną obsługę zaimplementować tutaj.